### PR TITLE
Suggest edits for RUM dashboards

### PR DESCRIPTION
**Pull request by billg-splunk**
Probably needs testing -- it works great apply and destroying, but I get errors on the destroy (even though it works).

### CODE REVIEW

**Comment by billg-splunk** [comment]
I have read the CLA and I hereby accept the Terms

**Comment by billg-splunk** [comment]
Recheck

**Comment by billg-splunk** [comment]
I have read the CLA Document and I hereby sign the CLA
I have read the Code of Conduct and I hereby accept the Terms

**Comment by greatestusername-splunk** [comment]
recheck

**Comment by billg-splunk** [comment]
I have read the Code of Conduct and I hereby accept the Terms

File: dashboards-and-dashboard-groups/RUM-Real-User-Monitoring/README.md [hunk 1]
```diff
@@ -7,5 +7,17 @@ This directory contains detail oriented dashboards and required chart definition
 
 Each of these dashboards is meant as a place to look at details of the specific metrics RUM provides split by Browser, App, or Rigor Synthetics.
 
-As usual with Terraform these scripts expect an environment variable named `signalfx_auth_token`
-  - Example: `export TF_VAR_signalfx_auth_token='this_is_my_api_token'`
\ No newline at end of file
+To use:
```
Comment (greatestusername-splunk) [inline dashboards-and-dashboard-groups/RUM-Real-User-Monitoring/README.md:10]: Thank you for the extra detail!

File: README.md [hunk 2]
```diff
@@ -17,6 +17,14 @@ Please see [CONTRIBUTING.md](./CONTRIBUTING.md) for details on contributing to t
 
 All contributors must sign the CLA and Code of Conduct. You will be prompted by the [cla-assistant](https://github.com/marketplace/actions/cla-assistant-lite) workflow action during your Pull Request for your agreement.
 
-To agree to the CLA and COC please comment these messages on your PR:
-- `I have read the CLA Document and I hereby sign the CLA`
-- `I have read the Code of Conduct and I hereby accept the Terms`
+To agree to the CLA and COC please comment these in **separate individual messages** on your PR:
```
Comment (greatestusername-splunk) [inline README.md:20]: Thank you for helping me troubleshoot this and for adding the clarification in!

**Review by greatestusername-splunk — APPROVED** [review]
Appreciate the contribution and thanks for the help with workflow!

Tested and working on my machine ;) 🖖

**Review by jadbSFx — APPROVED** [review]
LGTM

**Pull request merged**